apps/dashboard/src/utils/liquid-autocomplete.ts

```diff
@@ -108,8 +109,8 @@ export const completions =
         to: pos,
         options:
           matchingVariables.length > 0
-            ? matchingVariables.map((v) => createCompletionOption(v.label, 'variable'))
-            : variables.map((v) => createCompletionOption(v.label, 'variable')),
+            ? matchingVariables.map((v) => createCompletionOption(v.label, v.type ?? 'variable', v.boost, v.info))
```

Comment:
by default all the `v.type` will be variable, but for the `digest variables` the type is `digest` this then helps in targeting it in DOM for CSS manipulations
check editor.tsx